calculator.js

```diff
@@ -0,0 +1,119 @@
+(function() {
+
+    var inputField = document.getElementById('screen');
+    var currNumber = 0;
```

Comment:
Once you implement *Screen* properly, you will not need currNumber, since you will be able to retrieve it from *Screen*, by using a dedicated method.